charts/consul/values.yaml

```diff
@@ -129,7 +129,7 @@ global:
   # image that is used for functionality such as catalog sync.
   # This can be overridden per component.
   # @default: hashicorp/consul-k8s-control-plane:<latest version>
-  imageK8S: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.0.0-dev
+  imageK8S: hashicorp/consul-k8s-control-plane:1.0.0-beta1
```

Comment:
Could we update the consul image as well?

Reply:
+1